client/modules/User/components/SocialAuthButton.jsx

```diff
@@ -31,22 +42,35 @@ const StyledButton = styled(Button)`
   width: ${remSize(300)};
 `;
 
-function SocialAuthButton({ service }) {
+function SocialAuthButton({ service, link, connected }) {
```

Comment:
I spent a while thinking _does `link` mean that this button's an `href`_? What is it linking _to_? Maybe it's just pre morning-coffee.

Perhaps instead of `link`, just the "connected" prop is enough to change the label type. If `connected` is null then display the login label otherwise display the `connect/connected` labels?

Reply:
What I'm attempting to account for here is two different display options for the `SocialAuthButton`:
- One for the Login/Signup Page
- One for the Account Settings page, which uses the word "Link" rather than "Login". When the auth button is in "link" mode, it shows that a social account can either be "connected" or "not connected".